proxy/http/Http1ClientSession.cc

```diff
@@ -427,9 +427,12 @@ Http1ClientSession::release(ProxyClientTransaction *trans)
     ka_vio = this->do_io_read(this, INT64_MAX, read_buffer);
     ink_assert(slave_ka_vio != ka_vio);
 
-    // Y!?
-    // client_vc->add_to_keep_alive_lru();
-    client_vc->cancel_active_timeout();
+    if (client_vc) {
+      client_vc->add_to_keep_alive_queue();
+      if (client_vc) {
```

Comment:
Yes, that does seem easier.  I'll change to cancel first.  That should call no continuations so the client_vc field should not vanish on us.